data/utilities/broadband-internet/index.html

```diff
@@ -47,95 +47,101 @@
 <strong>Data Type:</strong> GIS Data Layer<br />
 <strong>Steward(s):</strong> AGRC & The Governor's Office Of Economic Development (GOED)</p>
 <p><strong>Abstract:</strong><br />
-This dataset contains the statewide availability of broadband internet in Utah. The data represents broadband providers' coverage areas, with their technology types and maximum advertised download and upload speeds. This mapping data is a product of the State of Utah Broadband Project.<br />
+This dataset contains the statewide availability of broadband internet in Utah. The data represents broadband providers' coverage areas, with their technology types and maximum advertised download and upload speeds. <br />
 <BR><br />
 <strong>Data Content:</strong><br />
 The following data is available for download:</p>
 <div class="grid package">
     <div class="grid__col grid__col--12-of-12">
-        <h3 id="WirelineBroadbandData">Wireline Broadband Data</h3>
+        <h3 id="BroadbandServiceData">Broadband Service Data</h3>
```

Comment:
Not that it is a big deal, but i wouldn't recommend changing these since people _can_ directly link to them. But I doubt anyone is.